python/huggingfaceserver/pyproject.toml

```diff
@@ -12,9 +12,9 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.9,<3.12"
 kserve = { path = "../kserve", extras = ["storage"], develop = true }
-transformers = "~4.40.2"
+transformers = "^4.40.2"
 accelerate = "~0.30.0"
-torch = "~2.3.0"
+torch = "2.3.0"
```

Comment:
any reason to lock down torch version ?

Reply:
My apologies, it was left over while I was testing some things while developing. Reverted it to `~2.3.0`